_posts/2020-03-11-sr2020-covid-19-updates.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: news
+title: COVID-19 Coronavirus Student Robotics Updates
+---
+
+As you will be aware, the COVID-19 situation is quickly evolving, having recently been delcared a pandemic. 
+Many large events are being cancelled and we want to be public about how we are responding to the latest news.
+
+## The Competition
+
+At the time of writing, we still intend to have the competition on the 18th and 19th April 2020, however are actively and carefully considering whether this needs to change.
+
+We will be contacting teams about our decision next week. We will also publish this on the Student Robotics website.
+
+Considering this is an event for minors, we will be taking decisions with an abundance of caution. 
+
+We are in contact with Reading University Students' Union who are continually reviewing the situation, and mitigating the risks associated with Coronavirus.
+
+### In the case of the competition going ahead
+
+We will be briefing all volunteers and attendees about best hygiene practices, both in advance and on the day, ensuring they're followed throughout the competition. 
+
+We expect everyone attending the competition to follow the [guidelines][phe-guidelines] on COVID-19 from Public Health England.
+
+### In the case of the competition being postponed
+
+We are considering weekends in late June / early July as alternatives for the competition. 
+We will be surveying team leaders to ensure we select the most appropriate date.
+
+## Tech Days
+
+At the time of writing, we still intend to run the London Tech Day on 28th March. However, we will contact all teams as soon as possible should this be cancelled.
+
+In the event of the competition being postponed, all planned Tech Days will be cancelled. 
+
+The [forum][forum] is still a great place to get help from blueshirts and other teams, in addition to showing off your progress!
+
+
+If you or your school has any questions about our handling of the event, please don't hesitate to contact us at teams@studentrobotics.org.
+
+We will keep this page updated with new information as it becomes available.
```

Comment:
nit: This is a news post, so will appear on both the homepage and in the news history. Is the plan that we'll add more new posts in future, or that we'll keep this one post up to date?

Perhaps clarify this here (and maybe link the "this page" or "this post" text to itself)?

Reply:
I think the idea was to continually update that news post instead of posting fresh ones

Reply:
Perhaps what we want is something slightly separate. A new isolated page with this information on, and a news post announcing that it exists?